ext/node/ops/crypto/cipher.rs

```diff
@@ -348,14 +406,22 @@ impl Decipher {
     input: &[u8],
     output: &mut [u8],
     auth_tag: &[u8],
+    auto_padding: bool,
   ) -> Result<(), AnyError> {
     use Decipher::*;
     match self {
-      Aes128Cbc(decryptor) => {
-        assert!(input.len() == 16);
-        let _ = (*decryptor)
-          .decrypt_padded_b2b_mut::<Pkcs7>(input, output)
-          .map_err(|_| type_error("Cannot unpad the input data"))?;
+      Aes128Cbc(mut decryptor) => {
+        if auto_padding {
+          assert!(input.len() == 16);
+          let _ = (*decryptor)
+            .decrypt_padded_b2b_mut::<Pkcs7>(input, output)
+            .map_err(|_| type_error("Cannot unpad the input data"))?;
+        } else {
+          assert!(input.len() % 16 == 0);
```

Comment:
A single assert `assert!(input.len() % 16 == 0);` outside the `if` is sufficient.